packages/net-stubbing/lib/server/middleware/request.ts

```diff
@@ -118,7 +118,13 @@ export const InterceptRequest: RequestMiddleware = async function () {
     throw new Error('req.body must be a string or a Buffer')
   }
 
-  if (getEncoding(req.body) !== 'binary') {
+  const encoding = getBodyEncoding(req)
+
+  debug('req.body encoding', encoding)
```

Comment:
hmm, this is a bit of a noisy debug log, i think it would make more sense to only log in the exceptional case (intercepting binary), and to also log the request url/ID if it matches

so like 
```suggestion
if (encoding === 'binary') {
  debug('req.body contained non-utf8 characters, treating as binary content %o', { requestId: request.id, req: _.pick(this.req, 'url') })
}
```